src/logger/debugContext.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * *Do not import this directly.* Instead, use the shortcut reference `logger.DebugContext`.
+ *
+ * Add debug contexts here. Although convention typically calls for enums ito
+ * be capitalized, for parity with the `LOG_DEBUG` env var, please use all
+ * lowercase.
+ */
+export const DebugContext = {
+  // e.g. swaps: 'swaps'
```

Comment:
was this meant to be completed while we start using it?

Reply:
Yeah this is just an example